node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/NodeRepositoryProvisioner.java

```diff
@@ -120,21 +120,6 @@ public NodeRepositoryProvisioner(NodeRepository nodeRepository, Zone zone,
         return asSortedHosts(preparer.prepare(application, cluster, nodeSpec, groups), resources);
     }
 
-    @Override
-    // TODO(mpolden): Remove
-    public void activate(NestedTransaction transaction, ApplicationId application, Collection<HostSpec> hosts) {
-        try (var lock = lock(application)) {
-            activate(hosts, new ActivationContext(0), new ApplicationTransaction(lock, transaction));
-        }
-    }
-
-    @Override
-    // TODO: Remove after November 2020
```

Comment:
I don't think this one can be removed yet?

Reply:
None of these are called from the config-model that I can tell. It uses `HostProvisioner` instead, which only has `prepare`.

Reply:
Ah, yes you're right.